### PR TITLE
parses and output comments

### PR DESCRIPTION
**Pull request by p0nce**
ability to list only signature
ability to select intrinsics by name or instruction sets ("ALL" supported)
display count of selected intrinsics, useful to know if some have been added
added a command-line and usage information
build on 32-bit
default output on stdout

By default the program will now do nothing and display help.

At a first approximation this is useful to extract list of intrinsics, a nice signature with comments etc
even if no implementation is provided, this can be useful.

### CODE REVIEW

**Comment by p0nce** [comment]
```
Running .\intrinsicripper.exe -n _mm_add_ps -b
/// Add packed single-precision (32-bit) floating-point elements in "a" and "b", and store the results in "dst".
__m128 _mm_add_ps(__m128 a, __m128 b) pure nothrow @nogc @safe
{
    /*
    FOR j := 0 to 3
        i := j*32
        dst[i+31:i] := a[i+31:i] + b[i+31:i]
    ENDFOR
        */
    //While parsing a Start an Exception was thrown.
    assert(false, "Failed to parse");
}


*** Seen 1 intrinsics.
```

File: source/app.d [hunk 1]
```diff
@@ -8,33 +8,150 @@ import html;
...
+        string inputXML = "data-3.4.4.xml";
+        string outputPath = null;
+
+        // selected sets is the union of selected instruction sets and selected names
+        string[] selectedTech = [];
+        string[] selectedNames = [];
+        bool help = false;
+        bool generateBodies = false;
+
+        for(int i = 1; i < args.length; ++i)
```
Comment (burner) [inline source/app.d:40]: please have a look at std.getopt

Comment (p0nce) [reply source/app.d:40]: well I don't see how is that a problem

Comment (burner) [reply source/app.d:40]: not a problem but you can get rid of half of the lines of code and can replace this with a very well tested piece of library.

**Pull request merged**